packages/components/psammead-byline/src/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { string, shape } from 'prop-types';
+import styled from 'styled-components';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+
+const AVATAR_DIAMETER = '64px';
```

Comment:
I believe we use REMs instead of pixels in Psammead, this would be 4rem.

Reply:
Thanks, Fahad. This was just a draft value used to demo the component in refinement today. It will change before I put this into review. 😄